src/Mount.cpp

```diff
@@ -664,6 +702,81 @@ bool Mount::connectToDriver( TMC2209Stepper* driver, const char *driverKind ) {
 #endif
 #endif
 
+/////////////////////////////////
+//
+// configureFocusdriver
+// TMC2209 UART only
+/////////////////////////////////
+#if (FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE) && (FOCUS_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART)
+#if SW_SERIAL_UART == 0
+  void Mount::configureFocusDriver(Stream *serial, float rsense, byte driveraddress, int rmscurrent, int stallvalue)
```

Comment:
You should refactor both instances of the serial into one, there's only a minor difference to the serial initialization

Reply:
Not in this PR. This pattern has been used for all the motors. We can revisit later, or once HAL is here.

Reply:
Fair, but duplication leads to copy+paste errors and missed patches. I always try to refactor in place where I'm already making changes